NobelLaureateDetails/Pages/Index.cshtml.cs

```diff
@@ -14,31 +14,37 @@ public JsonResult OnGet()
         {
             List<Laureate> filteredLaureates = new List<Laureate>();
 
-            using (WebClient webClient = new WebClient())
-            {
-                string PhysicsNinetyEight = webClient.DownloadString("http://api.nobelprize.org/v1/prize.json?category=physics&year=1998");
-                Prizes.Laureates Laureates = Prizes.Laureates.FromJson(PhysicsNinetyEight);
+            string physicsNinetyEightlaureates = getJsonFromUrl("http://api.nobelprize.org/v1/prize.json?category=physics&year=1998");
+            Prizes.Laureates Laureates = Prizes.Laureates.FromJson(physicsNinetyEightlaureates);
 
-                List<long> filteredIds = new List<long>();
+            List<long> laureatesIds = new List<long>();
 
-                foreach (Prizes.Prize prize in Laureates.PrizesArray)
-                {
-                    foreach (Prizes.Laureate laureate in prize.Laureates) {
-                        filteredIds.Add(laureate.Id);
-                    }
-                }
+            Prizes.Prize prize = Laureates.PrizesArray[1];
```

Comment:
Reduced complexity from O(m*n) to O(n)

Reply:
Ah, I urge caution here.  You've hardcoded this to the second element of the PrizesArray.  There are several issues with this:
1) You're not getting the first item.  That would be element 0.
2) You're assuming there is a second item.  If there is not anything there, this will throw an exception and crash the program.
3) You're not considering all of the prizes in the array.